client/js/widgets/label.js

```diff
@@ -36,6 +68,12 @@ export class Label extends Widget {
         this.input.style.paddingTop = 'unset';
       }
     }
+    if(delta.html !== undefined) {
+      if(delta.html)
+        setHTML(this);
```

Comment:
Unless you plan to use these functions for something else, they should be inside the class (like `applyDeltaToDOM`). You call them with `this.setHTML()` then and instead of `w` you can use `this` inside them.

Reply:
Originally I did, because I was doing something else wrong, but I deleted that part earlier.  I can work on fixing this.